docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -39,342 +37,123 @@ Devices with Intel® GNA support:
-   ``KEY_GNA_SW_FP32``           Executes the GNA-compiled graph on CPU but substitutes parameters and calculations from low precision to floating point (``FP32``).
-   ============================  ==============================================================================================================================================
+### Models caching
+Cache for GNA plugin may be enabled via common OpenVINO `ov::cache_dir` property due to import/export functionality support (see below).
 
-.. tab:: Python
+See [Model caching overview page](@ref openvino_docs_IE_DG_Model_caching_overview) for more details.
 
-   ========================  ==============================================================================================================================================
-   Mode                      Description
-   ========================  ==============================================================================================================================================
-   ``GNA_AUTO``              Uses Intel® GNA if available, otherwise uses software execution mode on CPU.
-   ``GNA_HW``                Uses Intel® GNA if available, otherwise raises an error.
-   ``GNA_SW``                *Deprecated*. Executes the GNA-compiled graph on CPU performing calculations in the same precision as the Intel® GNA, but not in the bit-exact mode.
-   ``GNA_SW_EXACT``          Executes the GNA-compiled graph on CPU performing calculations in the same precision as the Intel® GNA in the bit-exact mode.
-   ``GNA_HW_WITH_SW_FBACK``  Uses Intel® GNA if available, otherwise raises an error. If the hardware queue is not empty, automatically falls back to CPU in the bit-exact mode.
-   ``GNA_SW_FP32``           Executes the GNA-compiled graph on CPU but substitutes parameters and calculations from low precision to floating point (``FP32``).
-   ========================  ==============================================================================================================================================
+### Import/Export
+GNA plugin supports import/export capability which helps to significantly decrease first inference time. The model compile target is the same as the execution target by default. The default value for the execution target corresponds to available hardware, or latest hardware version supported by the plugin (i.e., GNA 3.0) if there is no GNA HW in the system. If you are willing to export a model for a specific version of GNA HW, please use ov::intel_gna::compile_target property.
 
-@endsphinxdirective
+### Stateful models
+GNA plugin natively supports stateful models.
 
-## <a name="supported-configuration-parameters">Supported Configuration Parameters</a>
+Please refer to [Stateful models] (@ref openvino_docs_IE_DG_network_state_intro) for more details about such models.
 
```

Comment:
```suggestion

Note that, typically, GNA is used in streaming scenarios, when minimizing the latency is important. Taking into account that POT does not support the Tensor Iterator layer, the recommendation is to use the `--LowLatency2` option of the Model Optimizer when converting an original model.
```

Reply:
Note that, typically, GNA is used in streaming scenarios, when minimizing the latency is important. Taking into account that POT does not support the Tensor Iterator layer, the recommendation is **to use the `--transform` option of the Model Optimizer to apply `LowLatency2`** transformation when converting an original model.